### PR TITLE
Es/smoke 2 flights

### PR DESCRIPTION
**Pull request by relaxing-observer**
PR with smoke test #2 from suit (flight search)

### CODE REVIEW

File: tests/data/smoke_tests_data.py [hunk 1]
```diff
@@ -0,0 +1,2 @@
+class Data:
+    flights_search_data = [("Berlin Brandenburg", "London Stansted", "2022-03-11", "2022-03-18", "Test", "User")]
```
Comment (yefim-mironyuk) [inline tests/data/smoke_tests_data.py]: No new line at end of file.

File: tests/pages/payment_page_locators.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    
+    PASSENGER_INFO_DROPDOWN = (By.CSS_SELECTOR, "div.passenger-details__toggle + icon")
+    PASSENGER_NAME = (By.CSS_SELECTOR, "span.passengers__name")
+    
+
+    CARD_DETAIL_FORM = (By.CSS_SELECTOR, "card-details > form")
+    BILLING_ADDRESS_FORM = (By.CSS_SELECTOR, "address-form > form")
+    USER_EMAIL = (By.CSS_SELECTOR, "div.contact-details__user-info div.b2")
+    PAY_BUTTON = (By.CSS_SELECTOR, "button.pay-button")
+
```
Comment (yefim-mironyuk) [inline tests/pages/payment_page_locators.py:19]: No new line at end of file.

File: utils/browser_helper.py [hunk 3]
```diff
@@ -116,3 +117,23 @@ def is_element_clickable(self, selector):
             return False
         logger.debug(f'Element "{selector}" is clickable')
         return True
+    
+    def choose_element_with_minimum_price(self, selector):
```
Comment (yefim-mironyuk) [inline utils/browser_helper.py]: "Choose element with minimum value" mb?

File: tests/conftest.py [hunk 4]
```diff
@@ -65,43 +68,64 @@ def pytest_runtest_makereport(item, call):
...
+    if password_cmd == "None":
+        password_cmd = DefaultCreds.PASSWORD
+    return password_cmd
+
 
 @pytest.fixture(scope="module")
 def main_page(browser):
     return MainPage(browser=browser, url=Urls.MAIN_PAGE_URL)
 
-
```
Comment (kit-91-daniil) [inline tests/conftest.py:73]: should be 2 blank lines I suppose

Comment (relaxing-observer) [reply tests/conftest.py:73]: agree, fixed

**Review by kit-91-daniil — APPROVED** [review]

**Review by TA-user — APPROVED** [review]

**Review by yefim-mironyuk — APPROVED** [review]

**Pull request merged**